rust/stracciatella/src/schemas/yaml/types/item-cursor.schema.yaml

```diff
@@ -0,0 +1,28 @@
+$schema: http://json-schema.org/schema#
+type: string
+title: Item Cursor
+description: The cursor when the item is used
+enum:
+  - INVALID
+  - QUEST
+  - PUNCH
+  - TARGET
+  - KNIFE
+  - AID
+  - TOSS
+  - MINE
+  - LPICK
+  - MDETECT
+  - CROWBAR
+  - SURVCAM
+  - CAMERA
+  - KEY
+  - SAW
+  - WIRECUT
+  - REMOTE
+  - BOMB
+  - REPAIR
+  - TRAJECTORY
+  - JAR
+  - TINCAN
+  - REFUEL
```

Comment:
This can now be used for `ubCursor` in `items.json` as well, right?

Reply:
Yes. I just didn't want to break existing mods that modify the `items.json`.